internal/k8s/converter.go

```diff
@@ -91,6 +93,14 @@ func NewUnstructuredConverter() (*UnstructuredConverter, error) {
 	projectcontour.AddKnownTypes(uc.scheme)
 	ingressroutev1.AddKnownTypes(uc.scheme)
 
+	// Add the core types we need
+	uc.scheme.AddKnownTypes(v1.SchemeGroupVersion, &v1.Secret{}, &v1.Service{})
```

Comment:
IIUC, `scheme.Scheme` has all the core types registered, so maybe we could use that.

Or we could just do what [client-go does](https://github.com/kubernetes/client-go/blob/master/kubernetes/scheme/register.go#L131) which would at least mean that we don't need to keep track of which core types we add.

Reply:
Ah, this was a holdover from troubleshooting another problem I had with the converter. I'll revert to `scheme.AddToScheme()`. Thanks.